utils/scraping.py

```diff
@@ -0,0 +1,53 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+SELENIUM_DRIVER = os.environ.get("SELENIUM_DRIVER")
+driver = webdriver.Firefox(executable_path=SELENIUM_DRIVER)
+
+
+def scrapeQuotes(url) -> list:
+    """This function scrape quotes data
+
+    Args:
+        url (str): link of the website to be scraped
+
+    Returns:
+        list: list of quotes data scraped
+    """
+
+    driver.get(url)
+
+    quotes_data = []
+    next_page = True
+
+    while next_page:
+
+        for quote_div in driver.find_elements(By.CLASS_NAME, "quote"):
+            quote = quote_div.find_element(By.CLASS_NAME, "text").text
+            author_name = quote_div.find_element(By.CLASS_NAME, "author").text
+            author_link = quote_div.find_element(By.TAG_NAME, "a").get_attribute("href")
+            tags = []
+
+            for tag in quote_div.find_elements(By.CLASS_NAME, "tag"):
+                tags.append(tag.text)
```

Comment:
Repetitive code, see if you can create a common utils file and add find_elements method there and use from that location.

Reply:
This is fine! Ignore!